File: CSharp/Algorithm/Greather.cs
using static System.Console;

public static class Program {
    public static void Main(string[] args) {
        Write("Digite um número: ");
        if (!int.TryParse(ReadLine(), out var a)) {
            Write("Deu erro");
            return;
        }
        Write("Digite outro número: ");
        if (!int.TryParse(ReadLine(), out var b)) {
            Write("Deu erro");
            return;
        }
        if (a > b) Write($"{a} é maior que {b}.");
        else if (b > a) Write($"{b} é maior que {a}.");
        else Write("Os dois números são iguais");
    }
}

//https://pt.stackoverflow.com/a/136298/101

File: JavaScript/Algorithm/Greather.js
function retornaMaiorValor() {
    var maior = 0;
    for (let i = 0; i < array.length; i++) if (array[i] > maior) maior = array[i];
    return maior;
}

let array = [2, 3, 6, 7, 10, 1];
console.log(retornaMaiorValor(array));

//https://pt.stackoverflow.com/q/436409/101
